tools/msvc/sbclang.targets

```diff
@@ -828,6 +814,7 @@
     <Link Condition="'$(UseWinObjCCRT)' == 'true'">
       <!-- Ensure we pick up operator new from our project rather than the C runtime -->
       <ForceSymbolReferences>??2@YAPAXI@Z</ForceSymbolReferences>
+      <AdditionalOptions>/DISALLOWLIB:libucrt.lib /DISALLOWLIB:libucrtd.lib /nodefaultlib:ucrt /nodefaultlib:ucrtd /nodefaultlib:vccorlib /nodefaultlib:msvcrt /nodefaultlib:vccorlibd /nodefaultlib:msvcrtd /defaultlib:winobjcrt winobjcrt.lib %(AdditionalOptions)</AdditionalOptions>
```

Comment:
no it can be I just didn't feel like doing it as input and wanted to make sure it was there. the important part is making sure no one else is a default lib. I'm going to leave it as is for now.